core/src/optimistic_confirmation_verifier.rs

```diff
@@ -218,7 +235,7 @@ mod test {
 
             // If root is on same fork, nothing should be returned
             optimistic_confirmation_verifier
-                .add_new_optimistic_confirmed_slots(optimistic_slots.clone());
+                .add_new_optimistic_confirmed_slots(optimistic_slots.clone(), None);
```

Comment:
Same here in all the `add_new_optimistic_confirmed_slots` calls in this test.  Let's pass-in the blockstore and verify the result at the end.

Reply:
done